godot-core/src/builtin/vectors/vector2i.rs

```diff
@@ -75,10 +100,15 @@ impl Vector2i {
     }
 
     /// Converts `self` to the corresponding `glam` type.
-    fn to_glam(self) -> glam::IVec2 {
+    fn to_glam(self) -> IVec2 {
```

Comment:
Please keep glam types qualified, so the code is more expressive.

Reply:
I've checked how other integer vector modules used glam types and they all omitted glam module. I'll include it for other integer modules too so it can be uniform.

Reply:
Makes sense, thanks!